voxel51/users/query.py

```diff
@@ -115,6 +115,20 @@ def add_search_or(self, field, search_strs):
         '''
         return self.add_search(field, "|".join(search_strs))
 
+    def add_generic_search(self, search_str):
+        '''Adds generic search, e.g. search across all searchable fields.
+
+        Overwritten if used in conjunction with other search functions.
```

Comment:
Why is this the case? I haven't looked at the backend implementation, but I don't see why you couldn't support:
```
.add_search("status", "COMPLETE")
.add_search_any_field("xxxx")
```
to match completed jobs that contain "xxxx" somewhere. It's just another `WHERE <condition> AND <other condition>` clause.

Reply:
Deprecated. Now possible.